src/services/params.ts

```diff
@@ -29,7 +29,7 @@ export function optional<TParam extends Param>(param: TParam): OptionalParamGetS
       return getParamValue(value, param)
     },
     set: (value) => {
-      if (!stringHasValue(value)) {
+      if (value === undefined) {
```

Comment:
Should we have a test for setting optional params?

Reply:
added ✅